### PR TITLE
Michael express lab01 complete

### PR DESCRIPTION
**Pull request by cdmichaelb**

### CODE REVIEW

File: michael/express/lab01/app.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const express = require("express"); // Import express
+const app = express(); // Create an instance of express
+
+// Defines port, optionally from environment variable
+const port = process.env.PORT || 3333;
```
Comment (AnthonyW90) [inline michael/express/lab01/app.js:5]: Love this!

File: michael/express/lab01/app.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+// Choose a random message from messages
+const randomMessage = () => {
+	return messages[Math.floor(Math.random() * messages.length)];
+};
+
+// Define Routes
+// Root route
+app.get("/", (req, res) => {
+	console.log("GET /");
```
Comment (AnthonyW90) [inline michael/express/lab01/app.js:26]: This is also a nice trick to see which route is being used

Comment (AnthonyW90) [reply michael/express/lab01/app.js:26]: We will add in an express addon to take care of this in the future too!

**Review by AnthonyW90 — APPROVED** [review]
Nice work! I really like the addition of your console logs notifying you of paths being visited. Also great idea using dotenv to keep track of the port.

File: michael/express/lab01/app.js (hunk 2 above)
Comment (cdmichaelb) [reply michael/express/lab01/app.js:26]: Oh, nice @ the addon. I should've looked for that in the first place probably.

I actually don't even know the best way to find node packages. But I suppose maybe this package? https://www.npmjs.com/package/express-route-logging

**Pull request merged**